tests/Aspire.StackExchange.Redis.Tests/AspireRedisExtensionsTests.cs

```diff
@@ -132,44 +132,51 @@ public void ConnectionNameWinsOverConfigSection(bool useKeyed)
             host.Services.GetRequiredKeyedService<IConnectionMultiplexer>("redis") :
             host.Services.GetRequiredService<IConnectionMultiplexer>();
 
-        Assert.Contains(AspireRedisHelpers.TestingEndpoint, connection.Configuration);
+        Assert.Contains(ConnectionString, connection.Configuration);
         // the connection string from config should not be used since it was found in ConnectionStrings
         Assert.DoesNotContain("unused", connection.Configuration);
     }
 
-    public static IEnumerable<object[]> AbortOnConnectFailData =>
-    [
-        [true, GetDefaultConfiguration(), false],
-        [false, GetDefaultConfiguration(), false],
-
-        [true, GetSetsTrueConfig(true), true],
-        [false, GetSetsTrueConfig(false), true],
-
-        [true, GetConnectionString(abortConnect: true), true],
-        [false, GetConnectionString(abortConnect: true), true],
-        [true, GetConnectionString(abortConnect: false), false],
-        [false, GetConnectionString(abortConnect: false), false],
-    ];
-
-    private static IEnumerable<KeyValuePair<string, string?>> GetDefaultConfiguration() =>
-    [
-        new KeyValuePair<string, string?>("ConnectionStrings:redis", AspireRedisHelpers.TestingEndpoint)
-    ];
+    [RequiresDockerTheory]
```

Comment:
These tests didn't need a server before - they were just a plain `[Theory]`. Can we revert these changes and keep what was there originally?

Reply:
Great catch, fixed.